docs/resources/projects/settings/project-setup.md

```diff
@@ -532,4 +532,127 @@ To resolve this, simply add a delay ([Wait](../../../resources/control-flow/time
 <p>
 We are aware of a limitation where widgets that are not visible on a page (i.e., you need to scroll down to see them) may not be highlighted. We are actively working to resolve this issue. As a temporary workaround, you can try placing the widget in an area that is visible without scrolling. We appreciate your patience and hope to have a fix soon!
 </p>
-</details>
\ No newline at end of file
+</details>
+
+## Dev Environments
+
+Dev environments in FlutterFlow allow you to set up multiple environments for your apps, such as **Development**, **Staging**, and **Production**. For each environment, you can create environment-specific variables, which can be used throughout your app to change its behavior based on the selected environment.
+
+For example, in an e-commerce app, you might define an `apiUrl` variable that points to different backends for Development, Staging, and Production. This allows you to test new payment gateways or inventory management features in a dev or staging environment. It ensures that backend updates are functioning correctly without affecting the live production environment, where real customer orders are processed.
+
+:::note
+By default, every FlutterFlow project starts with a Production environment, where real users interact with the data.
+:::
+
+
+:::info[When to Use Dev vs. Staging Environments]
+- **Dev Environment**: Use for testing and developing new features without affecting production data.
+- **Staging Environment**: Use to simulate the production environment before launching, and is isolated from the actual production data.
+
+
+*This is a common best practice, but you can create custom environments with different names for your own workflow style.*
+:::
+
+### Create and Switch Environments
+
+To create and switch environments, follow the steps below:
+
+<div style={{
```

Comment:
Got some reviews from some people that our Arcades restart as it ends without indicating that the end slide was completed which is confusing to users because sometimes they keep playing the Arcade only to realize they are re-doing the Arcade again. Can we ensure for these new Arcades and new Arcades that they dont automatically restart and let users restart it if needed?